avocado/plugins/jobs.py

```diff
@@ -116,15 +127,24 @@ def handle_list_command(jobs_results):
         for filename in jobs_results.values():
             with open(filename, "r", encoding="utf-8") as fp:
                 job = json.load(fp)
+
+                job_id = job.get("job_id", "N/A")
```

Comment:
You don't need to create new variables, you can you `job.get()` in the `LOG_UI.info`.